src/v2/recovery.ts

```diff
@@ -332,13 +333,11 @@ class CrossChainRecoveryTool {
     recoveryTx = recoveryTx || this.recoveryTx;
 
     // Determine fee with default fee rate
-    const P2SH_INPUT_SIZE = config.tx.P2SH_INPUT_SIZE;
-    const OUTPUT_SIZE = config.tx.OUTPUT_SIZE;
-    const TX_OVERHEAD_SIZE = config.tx.TX_OVERHEAD_SIZE;
     const feeRate = this.feeRates[this.sourceCoin.type];
 
     // Note that we assume one output here (all funds should be recovered to a single address)
-    const txSize = P2SH_INPUT_SIZE * recoveryTx.tx.ins.length + OUTPUT_SIZE + TX_OVERHEAD_SIZE;
+    const txSize = VirtualSizes.txP2shInputSize * recoveryTx.tx.ins.length + VirtualSizes.txP2pkhOutputSize +
```

Comment:
please check if this works, it should be more accurate: 

`Dimensions.fromInputs(tx.ins).plus(Dimensions.SingleOutput.p2pkh).getVSize()`

I also think we could be more accurate by using the actual output instead of guessing p2pkh

```
outputDims = Dimensions.fromOutputWithScriptLength(
  utxolib.address.toOutputScript(address, network).length
);
```

Reply:
https://bitgoinc.atlassian.net/browse/BG-11805